### PR TITLE
Cleaned up RewardManager contract

### PR DESCRIPTION
**Pull request by ith-harvey**
# Description of change
## High level
* Split `RewardsManager.t.sol` into -> `Rewards/RewardsManager.t.sol` & `Rewards/RewardDSTestPlus.sol`
* No changes to `src` were made

### CODE REVIEW

**Review by MikeHathaway — APPROVED** [review]
LGTM! Much cleaner like this

File: tests/forge/utils/DSTestPlus.sol [hunk 1]
```diff
@@ -8,6 +8,8 @@ import '@openzeppelin/contracts/token/ERC20/ERC20.sol';
 import '@openzeppelin/contracts/utils/structs/EnumerableSet.sol';
 
 import 'src/interfaces/pool/IPool.sol';
+import 'src/interfaces/rewards/IRewardsManager.sol';
```
Comment (prat-gpt) [inline tests/forge/utils/DSTestPlus.sol]: these are unused imports, can be removed.

**Review by prat-gpt — APPROVED** [review]
lgtm. added a comment.

File: tests/forge/utils/DSTestPlus.sol (hunk 1 above)
Comment (ith-harvey) [reply tests/forge/utils/DSTestPlus.sol]: Removed extra imports -> https://github.com/ajna-finance/contracts/pull/710/commits/006788d33d6ef5b9274ddfb9c6ca47d7974858c1

**Pull request merged**